README.md

```diff
@@ -2,38 +2,267 @@
 
-# Add signing and verification keys to the notation configuration policy
-notation key add --name $KEY_NAME --plugin azure-kv --id $KEY_ID
-notation cert add --name $KEY_NAME $CERT_PATH
+## Installation
+Before installing notation azure key vault plugin, please make sure Notation CLI has been installed. If you didn't install it, please follow the [Notation installation guide](https://notaryproject.dev/docs/installation/cli/).
 
-# Install ratify, with the verification key
-helm install ratify ratify/charts/ratify \
-        --set registryCredsSecret=regcred \
-        --set ratifyTestCert=$PUBLIC_KEY
-kubectl apply -f ./ratify/charts/ratify-gatekeeper/templates/constraint.yaml
+1. Navigate to the [Releases](https://github.com/Azure/notation-azure-kv/releases) page and select the latest release of notation-azure-kv. The notation-azure-kv binaries for each platform are available under the Assets section. Please download it based on your own platform. 
+2. Validate the checksum which should be the same as the one in `checksums.txt` and then install the plugin.
+   
+   For Linux Bash:
+   ```sh
+   # validate checksum
+   sha256sum notation-azure-kv_<version>_linux_amd64.tar.gz
 
-# Remotely sign with Azure Key Vault
-notation sign --key $KEY_NAME $IMAGE 
+   # install the plugin
+   mkdir -p "$HOME/.config/notation/plugins/azure-kv"
+   tar zxf notation-azure-kv_<version>_linux_amd64.tar.gz -C "$HOME/.config/notation/plugins/azure-kv" notation-azure-kv
+   ```
+   For MacOS Bash:
+   ```sh
+   # validate checksum
+   shasum -a 256 notation-azure-kv_<version>_linux_amd64.tar.gz
 
-# Deploy the image, with Gatekeeper, Ratify and Notary v2 validation of the signed image
-kubectl run net-monitor --image=$IMAGE -n demo
-```
+   # install the plugin
+   mkdir -p "$HOME/Library/Application Support/notation/plugins/azure-kv"
+   tar zxf notation-azure-kv_<version>_darwin_amd64.tar.gz -C "$HOME/Library/Application Support/notation/plugins/azure-kv" notation-azure-kv
+   ```
+   For Windows Powershell:
+   ```powershell
+   # validate checksum
+   (Get-FileHash .\notation-azure-kv_<version>_windows_amd64.zip).Hash
 
-See [documentation for details on remote signing with Azure Key Vault, validating a deployment to AKS with Notation and Ratify, using a simple setup script.](docs/nv2-bicep.md).
+   # install the plugin
+   mkdir "$env:AppData\notation\plugins\azure-kv"
+   Expand-Archive -Path notation-azure-kv_<version>_windows_amd64.zip -DestinationPath "$env:AppData\notation\plugins\azure-kv"
+   ```
+3. Try to run `notation plugin list` to show the installed plugin.
 
+## Getting started with a self-signed Azure Key Vault Certificate
+To demonstrate the basic use case, starting with a self-signed certificate is easier to understand how the Azure-kv plugin works with Notation, however, you **should not** use this in production because the self-signed certificate is not trusted.
```

Comment:
You may want to remove this line since self-signed certificates can be trusted depending on the scenarios.

Of course, you may replace it with a note:

> **Note** It is suggested to get a certificate from a trusted CA since self-signed certificate is not publicly trusted.